hedera-node/hedera-mono-service/src/test/resources/bootstrap/standard.properties

```diff
@@ -219,7 +219,7 @@ stats.speedometerHalfLifeSecs=10.0
 utilPrng.isEnabled=true
 tokens.autoCreations.isEnabled=true
 accounts.blocklist.enabled=true
-accounts.blocklist.resource=evm-addresses-blocklist.csv
+accounts.blocklist.path=evm-addresses-blocklist.csv
```

Comment:
same here , to avoid confusion provide path including `evm-addresses-blocklist.csv`